### PR TITLE
Refactor sagas

### PR DESCRIPTION
**Pull request by Tsaikkari**

### CODE REVIEW

File: src/components/JobseekerProfileForm/index.tsx [hunk 1]
```diff
@@ -33,35 +33,35 @@ const JobseekerProfileForm = () => {
...
+  //   phone,
+  //   seniority,
+  //   degreeName,
+  //   institutionName,
+  //   skills,
+  //   skillLevel,
+  //   duration,
+  // } = state
+
+  // const dispatch = useDispatch()
```
Comment (Wyfy0107) [inline src/components/JobseekerProfileForm/index.tsx]: Can you clean up the commented parts?

File: src/components/JobseekerProfileForm/index.tsx [hunk 2]
```diff
@@ -33,35 +33,35 @@ const JobseekerProfileForm = () => {
...
+    //   phone,
+    //   seniority,
+    //   degreeName,
+    //   institutionName,
+    //   skills,
+    //   skillLevel,
+    //   duration,
+    // }
+    // dispatch(updateJobseekerRequest(userData))
   }
```
Comment (Wyfy0107) [inline src/components/JobseekerProfileForm/index.tsx]: Here also

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

File: src/components/JobseekerProfileForm/index.tsx (hunk 2 above)
Comment (Tsaikkari) [reply src/components/JobseekerProfileForm/index.tsx]: Done.

**Review by Wyfy0107 — COMMENTED** [review]
Looks good 👍 But since we extract these data from action now, we can remove the unused state inside redux store

**Comment by Tsaikkari** [comment]
Removed the credential from store. I will make another pr for cleaned up store... :)

**Review by dilippoudel — APPROVED** [review]

File: src/components/JobseekerProfileForm/index.tsx (hunk 1 above)
Comment (Tsaikkari) [reply src/components/JobseekerProfileForm/index.tsx]: Done

**Pull request closed without merging**